### ISSUE TITLE
Allow TLS version used, to be discovered

### ISSUE THREAD

**Issue by MikeWhittakerRyff** [psf/requests#5499]
It's not possible to find which version of TLS was used, or indeed any of the SSL parameters, after a transfer. 
Could some of this metadata be made available, perhaps only if specified in the request.

Is there any indication that TLS was used at all ?

**Comment by sigmavirus24** [comment]
In the future, please search **closed and** open issues before creating new ones that are duplicates. This has been discussed ad nauseum and is blocked on urllib3 supporting it for which there's an open issue discussing the idea and implementation.

**Comment by MikeWhittakerRyff** [comment]
Apologies, thought I had searched in open issues, and could not find it in the manual, hence thought it was not a duplicate.

eg Curl does allow discovery of the SSL/TLS phases in a connection.

**Comment by nateprewitt** [comment]
Resolving as duplicate of previous tickets. We'll need work in urllib3 before we'd be able to consider exposing this in Requests.
